### ISSUE TITLE
2 ideas

### ISSUE THREAD

**Issue by katelynienaber** [mdanka/czech#55]
Hey! AWESOME APP!! I have been looking for something like this for ages ❤️ 🧡 💛 💚 💙 💜 

2 ideas:
- Could you add adjectives? Most useful (at least for me) would be to have adjectives and nouns together, so I could practice something like "modrá kniha, velké auto" etc. as one phrase...dunno how hard this would be
- Also some way to practice the prepositions with the cases would be cool...like "na" is used with accusative, "do" is used with genitive, etc...

Yeah anyway, super cool project, I hope you keep working on it ^^

**Comment by mdanka** [comment]
@katelynienaber Oh wow, I'm glad you like it :) These are definitely great ideas! I haven't had time to work on this unfortunately, but I hope I (or someone else) will get around to it soon enough! If you can contribute, I'd definitely be happy to help out.

**Comment by katelynienaber** [comment]
@mdanka Yes I'd be down to throw in, I won't promise to have time every month, but it sounds like you are not worried :)

So I could make a fork and open some pull requests? Would you maybe give me an overview of the code?

**Comment by mdanka** [comment]
That sounds great! Yeah, if you create pull requests, I'm happy to review
them.

As for the code, there are two parts:
* the database folder contains some scraper that builds a "database" (a
JSON) based on wiktionary pages (see this
<https://cs.wiktionary.org/wiki/P%C5%99%C3%ADloha:Frekven%C4%8Dn%C3%AD_seznam_(%C4%8De%C5%A1tina)/%C4%8CNK_SYN2005/1-1000>
for example)
* the src folder contains the main app, it's very simple

-Miklos
